src/components/ConnectToWallet/ConnectToWallet.tsx

```diff
@@ -153,126 +157,128 @@ const ConnectToWallet: React.FC<ConnectToWalletProps> = ({ onConnect }) => {
 	};
 
 	return (
-		<div className={`${WalletStyles.connect} border-primary`}>
-			<div className={WalletStyles.header}>
-				<h3 className={`glow-text-white`}>Connect To A Wallet</h3>
-			</div>
-			<hr className="glow" />
+		<Overlay open centered onClose={closeModal}>
+			<div className={`${WalletStyles.connect} border-primary`}>
+				<div className={WalletStyles.header}>
+					<h3 className={`glow-text-white`}>Connect To A Wallet</h3>
+				</div>
+				<hr className="glow" />
 
-			{isLoading && (
-				<div className={WalletStyles.Disconnect}>
-					<hr className="glow" />
-					<FutureButton glow onClick={() => {}}>
-						<div
-							style={{
-								display: 'flex',
-								justifyContent: 'center',
-								verticalAlign: 'center',
-								alignItems: 'center',
-							}}
-						>
+				{isLoading && (
+					<div className={WalletStyles.Disconnect}>
+						<hr className="glow" />
+						<FutureButton glow onClick={() => {}}>
```

Comment:
`FutureButton ` has props `onClick?: (event?: any) => void;`, so we can just use `<FutureButton glow>`